src/hotspot/share/runtime/stackOverflow.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 1997, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You have this copyright as a range of years while stackOverflow.hpp is just 2020.

Reply:
oops. fixed.